recipes/pcapplusplus/22.11/conanfile.py

```diff
@@ -0,0 +1,124 @@
+from conan import ConanFile
```

Comment:
Is there a particular reason you added a new version recipe? General we try to have only 1 unless there's a huge difference...

I I paste https://raw.githubusercontent.com/conan-io/conan-center-index/96cfead04e245ba4243da176b07a1e4a879f6cef/recipes/pcapplusplus/22.05/conanfile.py over this file the diff only shows me 
![image](https://user-images.githubusercontent.com/16867443/205160262-da786323-897a-48d2-a5c2-19d21532cf00.png)


Did you consider only using one recipe?

Reply:
Thanks @prince-chrismc for your response!

As you can see from the diff file, the new version doesn't need `pthreads` as a dependency and it also needs fewer patches. I though it makes more sens to add a recipe than adding a lot of `if` statements in the current recipe. Please let me know what you think

Reply:
These are super close I think it can be just 1 recipe. It's just a few lines versus an entire recipe

The patches are listed in the conandata per version so that should be easy too

Reply:
I'd greatly appreciate if you could combine the two versions

Reply:
ok sure, I'll fix it 👍

Reply:
@prince-chrismc I opened a new PR with the fix: https://github.com/conan-io/conan-center-index/pull/14712

Can you review it?